client/gulpfile.js

```diff
@@ -67,6 +103,7 @@ gulp.task('watch', () => {
   })
   gulp.watch(sources.css.all, ['styles', 'stylelint'])
   gulp.watch(sources.scripts.all, ['scripts'])
+  gulp.watch(sources.images.svg.all, ['svgstore'])
```

Comment:
Okay - just wondering why we're making SVGs a subset of images, and not a asset type of it's own.

Just a question.